Cargo.toml

```diff
@@ -175,7 +175,11 @@ zeroize = { opt-level = 3 }
 [profile.release]
 # Polkadot runtime requires unwinding.
 panic = "unwind"
-opt-level = 3
+# opt-level = 3
+opt-level = "z"
```

Comment:
There is no need. We don't need to optimize for size, optimizing for speed is better here. The machines that run Polkadot are not size restricted.